zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/DependencyResolver.java

```diff
@@ -177,8 +177,9 @@ public synchronized void copyLocalDependency(String srcPath, File destPath)
             DependencyFilterUtils.andFilter(exclusionFilter, classpathFilter));
     try {
       return system.resolveDependencies(session, dependencyRequest).getArtifactResults();
-    } catch (NullPointerException ex) {
-      throw new RepositoryException(String.format("Cannot fetch dependencies for %s", dependency));
+    } catch (Exception ex) {
```

Comment:
Good question. I  changed it to Exception as I see that org.sonatype.aether.RepositorySystem can throw DependencyResolutionException and I suspect we can have file exceptions and other in the whole org.sonatype.aether library. But because I didn't actually get an error here, I'll leave the final decision to you. Should we revert this to only have NPE or keep it broad with Exception?